test/Microsoft.Azure.Devices.Edge.Test/Microsoft.Azure.Devices.Edge.Test.csproj

```diff
@@ -31,14 +30,18 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <ProjectReference Include="..\Microsoft.Azure.Devices.Edge.Test.Common\Microsoft.Azure.Devices.Edge.Test.Common.csproj" />
+    <PackageReference Include="Microsoft.Extensions.Configuration.Binder" Version="2.2.4" />
+    <PackageReference Include="Microsoft.Extensions.Configuration.EnvironmentVariables" Version="2.2.4" />
+    <PackageReference Include="Microsoft.Extensions.Configuration.Json" Version="2.2.0" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.9.0" />
+    <PackageReference Include="nunit" Version="3.11.0" />
```

Comment:
So have we settled on Nunit for our e2e tests?

Reply:
Yes. I'll be keeping an eye on this throughout, to see if it makes sense to switch. But unfortunately xUnit (as well as mstest) does not support real-time console output. Because of how long these tests take (and how impatient I am while they're running), it's been very helpful to see progress during the tests.